environment.yaml

```
@@ -15,6 +16,8 @@ dependencies:
 - jupyterlab
 - jupyterlab-spellchecker
 - nebari-dask
+- panel
+- hvplot
```

```suggestion
- hvplot
- streamlit
- cdsdashboards-singleuser
```
`cdsdashboards-singleuser` is required for this env to appear as an option in CDS dashboards drop down.

---

I'm updating this env now.